src/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -17,6 +17,14 @@
 #include <ngraph/pass/manager.hpp>
 #include <ngraph/type/bfloat16.hpp>
 #include <ngraph/pass/serialize.hpp>
+#include "ngraph/opsets/opset1.hpp"
+#include "ngraph/opsets/opset3.hpp"
+#include "ngraph/opsets/opset4.hpp"
+#include "ngraph/opsets/opset5.hpp"
+#include "ngraph/opsets/opset6.hpp"
+#include "ngraph/opsets/opset7.hpp"
+#include "ngraph/opsets/opset8.hpp"
+#include "ngraph/opsets/opset9.hpp"
```

Comment:
I'm afraid it will affect compilation time because of multiple heavy headers in common test header files..

It's better to include concrete ops headers into .cpp files where operations are used

Reply:
Deleted all headers, applied required changes inplace